config/testnet/exceptions.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
 		'16272500600161825502', // 336424
 		'17197328760149985951', // 341635
 		'18231026627962552928', // 917323
+		'13238495338830587861', // 1305933
```

Comment:
According to changes made in https://github.com/LiskHQ/lisk/pull/2442/files this exception is no longer a valid one.

Reply:
@SargeKhan discussed in the morning, this exception is required. @SargeKhan Can you confirm.

Reply:
Ok, let's open a separate PR to add a valid exception connected with fix #2434